numpy/_core/fromnumeric.py

```diff
@@ -3755,15 +3776,33 @@ def var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue, *,
 
     Notes
     -----
-    The variance is the average of the squared deviations from the mean,
-    i.e.,  ``var = mean(x)``, where ``x = abs(a - a.mean())**2``.
-
-    The mean is typically calculated as ``x.sum() / N``, where ``N = len(x)``.
-    If, however, `ddof` is specified, the divisor ``N - ddof`` is used
-    instead.  In standard statistical practice, ``ddof=1`` provides an
-    unbiased estimator of the variance of a hypothetical infinite population.
-    ``ddof=0`` provides a maximum likelihood estimate of the variance for
-    normally distributed variables.
+    The (sample) variance is an average of the squared deviations from the
+    sample mean.
+
+    The squared deviations are calculated as ``x2 = abs(a - a.mean())**2``,
+    and the average is taken as ``x2.sum() / (N - ddof)``, where ``N`` is the
+    number of observations in the sample `a` and `ddof` is a keyword parameter.
+    This is a "consistent" estimator of the variance of the population from
+    which the sample was drawn: as the number of observations tends toward
+    infinity, it converges in probability to the true population variance.
+
+    With ``ddof=0`` (default), the resulting sample variance is known as the
+    "uncorrected sample variance" because it is not corrected for "bias": the
+    expected value of this sample variance does not equal the true population
+    variance. Nonetheless, it is a useful estimator of the population variance
+    when the sample size is large, and it is the maximumum likelihood estimate
+    of the variance when the population is normally distributed.
+
+    Use of ``ddof=1`` is known as the "Bessel correction", and the resulting
+    "corrected sample variance" (or simple *the* "sample variance") is an
+    unbiased estimator of the population variance. Other values of `ddof` may
+    have desirable properties for samples drawn from certain distributions;
+    for instance, if the population is normally distributed, ``ddof=1.5``
+    nearly eliminates the bias. Unfortunately, there is no simple, unbiased
+    estimator of the population variance that is valid for all distributions,
+    and reducing the bias often *increases* the mean squared error between the
+    sample variance and population variance. Consequently, the "best" choice of
+    `ddof` depends on the use case and is often subjective.
```

Comment:
@eendebakpt How about simplifying it to:

```suggestion
    "corrected sample variance" (or simply *the* "sample variance") is an
    unbiased estimator of the population variance. However,
    reducing the bias often *increases* the mean squared error between the
    sample variance and population variance. Consequently, the "best" choice of
    `ddof` depends on the use case and is often subjective.
```